testing-common/src/main/java/io/opentelemetry/instrumentation/testing/util/ContextStorageCloser.java

```diff
@@ -43,6 +43,7 @@ private static void cleanup(AutoCloseable storage) throws Exception {
         .ignoreException(AssertionError.class)
         .atMost(Duration.ofSeconds(10))
         .pollInterval(Duration.ofSeconds(1))
+        .pollDelay(Duration.ZERO)
```

Comment:
apparently poll delay is set to poll interval by default which will just slow down the build